transactions/manager.go

```diff
@@ -12,38 +12,69 @@ import (
 	"github.com/satori/go.uuid"
 )
 
-// Manager wraps the repository and exposes specific functions.
-type Manager interface {
+// Config is the config interface for transactions package
+type Config interface {
+	GetEthereumContextWaitTimeout() time.Duration
+}
 
+// Manager is a manager for centrifuge transactions.
+type Manager interface {
 	// ExecuteWithinTX executes a transaction within a transaction
 	ExecuteWithinTX(ctx context.Context, accountID identity.CentID, existingTxID uuid.UUID, desc string, work func(accountID identity.CentID, txID uuid.UUID, txMan Manager, err chan<- error)) (txID uuid.UUID, done chan bool, err error)
-
-	// CreateTransaction
-	// TODO [TXManager] remove this once TX Manager update is complete
-	CreateTransaction(accountID identity.CentID, desc string) (*Transaction, error)
 	GetTransaction(accountID identity.CentID, id uuid.UUID) (*Transaction, error)
-	SaveTransaction(tx *Transaction) error
+	UpdateTaskStatus(accountID identity.CentID, id uuid.UUID, status Status, taskName, message string) error
```

Comment:
This is a key txManager update in this pr, we use task status instead of tx status to track progress now in tasks